qemu/tests/timerdevice_check_ntp_offset.py

```diff
@@ -0,0 +1,96 @@
+import os
+import time
+import logging
+
+from avocado.utils import process
+from virttest import utils_test
+from virttest import data_dir
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check the offset of Meinberg NTP for windows guest.
+
+    1) sync the host time with ntp server
+    2) boot a windows  guest with network
+    3) install diskspd tool and Meinberg NTP
+    4) run diskspd benchmark in Administrator user
+    5) play a video fullscreen
+    6) periodically verify "offset" output of Meinberg NTP
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+
+    def scp_file(file_dir, file_name, file_dst_path):
+        file_src_path = os.path.join(data_dir.get_deps_dir(), file_dir, file_name)
```

Comment:
`get_deps_dir(file_dir)`

Reply:
And, i think this function is not necessary, how do you think about use `vm.copy_files_to` directly.